app/components/standup_meeting_group/joined_card_component.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= tag.div(id: dom_id(standup_meeting_group), class: "card card-bordered w-72") do %>
+  <div class="card-body flex flex-row p-4">
+    <div class="w-1/2">
+      <h4 class="card-title font-bold text-lg"><%= link_to standup_meeting_group.name, standup_meeting_group_path(standup_meeting_group.id) %></h4>
+    </div>
+    <div class="w-1/2">
+      <div class="card-actions flex space-x-2 justify-end">
+        <button class="btn btn-sm">Check In</button>
+        <button class="btn btn-sm">Dismiss</button>
+        <br>
+        <%= link_to 'Leave', standup_meeting_group_join_path(@standup_meeting_group, @standup_meeting_group_user), data: { turbo_method: :delete }, class: 'btn btn-link align-bottom' %>
```

Comment:
It looks like they've pulled leaving off the card in the wireframes. Maybe worth asking Daniel or Ariel about this and if they still want it here.

Reply:
hmm, yes I see. I'll post a question in discord